lesson_2_Section_30_new.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>If you extend to all X you'd get a flat line that's just the average of everything, right?  Or are you doing like a sliding window?  I think it makes it super clear that, one data point on either side isnt enough data points, all X is too many data points (flat line), how do you pick the right number?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>